README.md

```diff
@@ -235,7 +258,165 @@ as Notepad.
 2. Download the `TR1X-Installer.dmg` installer image. Mount the image and drag TR1X to the Applications folder.
 3. Run TR1X from the Applications folder. This will show you an error dialog about missing game data files. This is expected at this point, as you have not copied them in yet. However, it's important to run the app first to allow macOS to verify the app bundle's signature.
 4. Find TR1X in your Applications folder. Right-click it and click "Show Package Contents".
-5. Copy your Tomb Raider 1 game data files into `Contents/Resources/data`. (See the Windows / Linux instructions for retrieving game data from e.g. GOG.)
+5. Copy your Tomb Raider 1 game data files into `Contents/Resources`. (See the Windows / Linux instructions for retrieving game data from e.g. GOG.)
+
+If you install everything correctly, your game directory should look more or
+less like this (click to expand):
+
+<details>
+<p><em>* Will not be present until the game has been launched.</em></p>
+<pre>
+.
+└── Contents
+    ├── _CodeSignature
+    ├── Framworks
+    ├── info.plist
+    ├── MacOS
+    └── Resources
+        ├── cfg
+        │   ├── TR1X.json5 *
+        │   ├── TR1X_gameflow.json5
+        │   ├── TR1X_gameflow_demo_pc.json5
+        │   └── TR1X_gameflow_ub.json5
+        ├── data
+        │   ├── cat.phd
+        │   ├── cred0.pcx
+        │   ├── cred1.pcx
+        │   ├── cred2.pcx
+        │   ├── cred3.pcx
+        │   ├── cut1.phd
+        │   ├── cut2.phd
+        │   ├── cut3.phd
+        │   ├── cut4.phd
+        │   ├── egypt.phd
+        │   ├── eidospc.pcx
+        │   ├── eidospc.png
+        │   ├── end2.phd
+        │   ├── end.pcx
+        │   ├── end.phd
+        │   ├── gym.phd
+        │   ├── install.pcx
+        │   ├── level10a.phd
+        │   ├── level10b.phd
+        │   ├── level10c.phd
+        │   ├── level1.phd
+        │   ├── level2.phd
+        │   ├── level3a.phd
+        │   ├── level3b.phd
+        │   ├── level4.phd
+        │   ├── level5.phd
+        │   ├── level6.phd
+        │   ├── level7a.phd
+        │   ├── level7b.phd
+        │   ├── level8a.phd
+        │   ├── level8b.phd
+        │   ├── level8c.phd
+        │   ├── titleh.pcx
```

Comment:
I'd add `titleh_ub.pcx` under `titleh.pcx` because this file is part of the collection.